lib/src/main/kotlin/com/github/bratek20/hla/directory/impl/DirectoriesLogic.kt

```diff
@@ -11,7 +11,7 @@ class DirectoriesLogic: Directories {
             "File does not exist or not dir for path: $nioPath"
         }
 
-        val allFiles = file.listFiles()
+        val allFiles = file.listFiles()?.sorted()
```

Comment:
lets add tests for Directory module showing example of expected sorting

Reply:
added

Reply:
wrong place

Reply:
moved in correct file